src/node.rs

```diff
@@ -616,51 +622,93 @@ impl PbftNode {
                     .get_unvalidated_block_with_id(block.previous_id.as_slice())
             });
         if previous_block.is_none() {
-            self.service
-                .fail_block(block.block_id.clone())
-                .unwrap_or_else(|err| error!("Couldn't fail block due to error: {:?}", err));
-            return Err(PbftError::InternalError(format!(
-                "Received block {:?} / {:?} but node does not have previous block {:?}",
-                block.block_num,
-                hex::encode(&block.block_id),
-                hex::encode(&block.previous_id),
-            )));
-        }
+            //It means that the block has come out of sequence .  We should not fail the  block,
```

Comment:
Please update the formatting of all comments to match the existing comments in the codebase. Should have a space after `//`, begin with a capital letter, remove extra spaces, and wrap comments at 100 characters.

Reply:
Noted. Thanks.